NimGame/Program.cs

```diff
@@ -0,0 +1,52 @@
+﻿//Start of the game
+
+Gamestart:
+
+Console.WriteLine("Welcome to Nim");
+Console.WriteLine("|||||||||||||||||||||||| (24)");;
+
+int totalMatches = 24;
+bool playerTurn = true;
+
+matches:
+    Console.WriteLine("How many matches do you want to draw?");
+    int userInput = int.Parse(Console.ReadLine());
+
+
+//Maximum matches you can pick is 3.
+if (userInput is >3 or < 1)
+{
+    Console.WriteLine("Trying to cheat ehh, you can only pick between 1 - 3, Pick Again!");
+    goto matches;
+}
+
+if (userInput >= 0 && totalMatches <= 24);
```

Comment:
Can be removed.
Edit: You're also lucky that you accidentally put a semicolon after the if